CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 CHANGELOG
 =========
 
+3.x.x
```

Comment:
The next release will be 3.6.1, so I suggest to specify that to avoid misunderstandings